### DISCUSSION TITLE
Save Option for Domination Arma 2

### DISCUSSION THREAD

**Discussion by Tr8tor** [Xeno69/Domination#465] [Q&A]
Hi Xeno,

  Loved your domination since first Arma, great work.
  
  I've booted up Arma 2 out of nostalgia recently, and I actually enjoy it more than Arma 3. I also had saved some MP missions including your domination ( co40 Domination! West 2.71 ), and naturally I launched a lone LAN session. It feels great, very nostalgic, running it with some mods too, for improved gameplay. But there's a small issue, the mission does not support saving like the vanilla LAN warfare missions. Is it possible to enable saving for your domination mission for Arma 2?

**Comment by Xeno69** [comment]
No, not possible.

The mission was never designed to be used with the game saving mechanism.
